teachers_digital_platform/css/organisms/expandable.less

```diff
@@ -0,0 +1,7 @@
+// Expandable header
+
+.o-expandable_header-left {
+    .tdp-activity-search & {
```

Comment:
I would suggest avoiding this nesting when there is nothing else being nested inside `.o-expandable_header-left`. A simple `.tdp-activity-search .o-expandable_header-left` selector would be more straightforward.

Reply:
This makes sense.  Fixed.